lib/src/model.g.dart

```diff
@@ -15,7 +15,7 @@ Summary _$SummaryFromJson(Map<String, dynamic> json) => new Summary(
     json['packageName'] as String,
     json['packageVersion'] == null
         ? null
-        : new Version.parse(json['packageVersion']),
+        : new Version.parse(json['packageVersion'] as String),
```

Comment:
Did you make these changes by hand?

Reply:
Updated the `tool/src/version_helper.dart`